src/core/sys/posix/iconv.d

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+
+    D binding for the POSIX iconv library.
+
+    See_Also:
+       http://pubs.opengroup.org/onlinepubs/009695399/functions/iconv_open.html
+
+*******************************************************************************/
+
+module core.sys.posix.iconv;
+
+version (Posix):
+extern (C):
```

Comment:
BTW,

> This function is a possible cancellation point and therefore not marked with __THROW.  

no need to worry about that. Other functions that are cancellation points are still marked with `nothrow` in druntime. There would be no point in not marking them with `nothrow` because `abi::__forced_unwind` exceptions are not meant to be caught anyway (I'm not even sure if it is possible to catch them in D).